src/common/veteran.js

```diff
@@ -67,13 +67,13 @@ const blankVeteran = {
     year: makeField(''),
   },
   dateOfMarriage: {
-    month: makeField(''),  // TODO(awong): Ignored by ES System
-    day: makeField(''),  // TODO(awong): Ignored by ES System
-    year: makeField('')  // TODO(awong): Ignored by ES System
+    month: makeField(''),
+    day: makeField(''),
+    year: makeField('')
   },
   sameAddress: makeField(''),  // TODO(awong): Not sure how to handle the mapping.
-  cohabitedLastYear: makeField(''),  // TODO(awong): This should be scoped to spouse.
-  provideSupportLastYear: makeField(''),  // TODO(awong): This should be scoped to spouse.
+  cohabitedLastYear: makeField(''),  // TODO(awong): This name should be scoped to spouse.
```

Comment:
I consider this part of validation + testing as I think unittesting will make us really want to refactor the object structure a little.